### DISCUSSION TITLE
Install dependency from source instead pip or apt

### DISCUSSION THREAD

**Discussion by carlosecv** [Tecnativa/doodba#427] [Doodba - The template] [answered]
I have a problem with conekta library installed by pip im getting a problem of indentation on library python code, so, is there a way to install a library on doodba from github repository instead pip or apt or maybe another way?

The isssue is from 
File "/usr/local/lib/python3.5/site-packages/conekta/__init__.py", line 629
    return "orders"

Or

How can i go to this folder and modify the __init__.py file because its mounted readonly?

**Comment by carlosecv** [comment, marked as answer]
I found the solution:

On pip.txt instead of putting the name of the library
conekta
we can fork the original repository and then add it here as:
git+https://github.com/myrepo/conekta-python.git@master

Then rebuild the containers

**Reply by joao-p-marques** [reply]
Ok, that is an option. Alternatively, you can remove it from `pip.txt` and instead use a build script in the `build.d` folder. There, you can put a bash or python script with whatever you need and it will be executed at build time. You can find out more [here](https://github.com/Tecnativa/doodba#optodoocustombuildd).
